### PR TITLE
Update react-pdf to 1.6.1 Closes #1533

### LINKED ISSUE #1533: Remove warnings due to the react update
Dependencies to update:
- [x] react-router
- [x] react-spinkit
- [ ] react-pdf
Fixes to do:
- [x] event cancelBubble is deprecated

### PR DESCRIPTION
**Pull request by Gnafu**

### CODE REVIEW

File: web/client/components/print/PrintOption.jsx [hunk 1]
```diff
@@ -28,7 +28,7 @@ const PrintOption = React.createClass({
         };
     },
     onChange() {
-        this.props.onChange(this.refs.input.getInputDOMNode().checked);
+        this.props.onChange(!this.refs.input.props.checked);
```
Comment (mbarto) [inline web/client/components/print/PrintOption.jsx]: Are you inverting the value?

File: web/client/components/TOC/background/BackgroundSwitcher.jsx [hunk 2]
```diff
@@ -101,7 +101,7 @@ let BackgroundSwitcher = React.createClass({
     },
     render() {
         return (
-           <Grid id={this.props.id} className="BackgroundSwitcherComponent" header={this.props.name} fluid={this.props.fluid}>{this.renderBackgrounds()}</Grid>
+           <Grid id={this.props.id} className="BackgroundSwitcherComponent" fluid={this.props.fluid}>{this.renderBackgrounds()}</Grid>
```
Comment (mbarto) [inline web/client/components/TOC/background/BackgroundSwitcher.jsx]: Is header not needed anymore?

File: web/client/components/print/__tests__/PrintPreview-test.jsx [hunk 3]
```diff
@@ -48,42 +48,34 @@ describe("Test the PrintPreview component", () => {
         expect(node).toExist();
     });
 
-    it('creates component and loads pdf', (done) => {
```
Comment (mbarto) [inline web/client/components/print/__tests__/PrintPreview-test.jsx]: Why did you remove this test?

File: web/client/components/print/PrintOption.jsx (hunk 1 above)
Comment (Gnafu) [inline web/client/components/print/PrintOption.jsx]: Yes, the old one.
getInputDOMNode() just returns null

File: web/client/components/TOC/background/BackgroundSwitcher.jsx (hunk 2 above)
Comment (Gnafu) [inline web/client/components/TOC/background/BackgroundSwitcher.jsx]: I don't think it eve was, header is not a property of Grid

File: web/client/components/print/__tests__/PrintPreview-test.jsx (hunk 3 above)
Comment (Gnafu) [inline web/client/components/print/__tests__/PrintPreview-test.jsx]: It was testing ReactJS and it is no longer mockable
